learning_observer/learning_observer/auth/social_sso.py

```diff
@@ -210,6 +239,35 @@ async def _process_student_documents(student):
             await _process_student_documents(student)
     # TODO saved skipped doc ids somewhere?
 
+async def _canvas(request):
+    '''
+    Handle Canvas authorization
+    '''
+    if 'error' in request.query:
+        return {}
+    
+    default_server = settings.pmss_settings.default_server(types=['lms', 'canvas_oauth'])
+    
+    url = f'https://{default_server}/login/oauth2/token'
```

Comment:
Notice that with Google, we specify the entire URI in creds as opposed to hard coding the end here. We should remain consistent. 
Is there a reason that it should be hardcoded like this instead?

Reply:
I renamed to `token_uri` to look like Google's own